airflow/providers/docker/CHANGELOG.rst

```diff
@@ -27,6 +27,8 @@
 Changelog
 ---------
 
+* ``Use the 'DOCKER_HOST' environment variable as the default value for the 'docker_url' parameter (#38387).``
```

Comment:
By note here i meant not the title of the PR (this will indeed be automatically retrieved) but a hand-written note on what to expect if you are user, explaining potential problems and remediations. Something like 

https://github.com/apache/airflow/blob/main/airflow/providers/google/CHANGELOG.rst#L113 for example

Reply:
Oops, I misunderstood; thanks for your patience. I updated it.

All CI checks that weren't skipped were green. :rocket: